client/share_trader/js/components/share_trader.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-
+import 'core-js/stage/4';
```

Comment:
Put this in the `app.js`, and I guess we might as well include it in each of the `app.` to ensure language feature parity between all of them.